src/coord/config.c

```diff
@@ -51,15 +51,27 @@ CONFIG_GETTER(getClusterTimeout) {
 }
 
 CONFIG_SETTER(setGlobalPass) {
+  RedisModule_Log(RSDummyContext, "warning",
+    "OSS_GLOBAL_PASSWORD is deprecated. Use `CONFIG SET search-oss-global-password <password>` instead");
```

Comment:
Need a documentation PR for the deprecation?

Reply:
https://redis.io/docs/latest/develop/interact/search-and-query/basic-constructs/configuration-parameters/#oss_global_password

Reply:
Will be added later (DOC-4649)